src/constants/MarkdownFiles/posts/2025-07-13-gsoc-25-MebinThattil-week06.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "GSoC ’25 Week 06 Update by Mebin J Thattil"
+excerpt: "Optimizations and reading documentation"
+category: "DEVELOPER NEWS"
+date: "2025-07-13"
+slug: "2025-07-13-gsoc-25-mebinthattil-week6"
+author: "@/constants/MarkdownFiles/authors/mebin-thattil.md"
+tags: "gsoc25,sugarlabs,week06,mebinthattil,speak_activity"
+image: "assets/Images/GSOCxSpeak.png"
+---
+
+<!-- markdownlint-disable -->
+
+# Week 06 Progress Report by Mebin J Thattil
+
+**Project:** [Speak Activity](https://github.com/sugarlabs/speak)  
+**Mentors:** [Chihurumnaya Ibiam](https://github.com/chimosky), [Kshitij Shah](https://github.com/kshitijdshah99)  
+**Assisting Mentors:** [Walter Bender](https://github.com/walterbender), [Devin Ulibarri](https://github.com/pikurasa)  
+**Reporting Period:** 2025-07-07 - 2025-07-13
+
+---
+
+## Goals for This Week
+
+- **Goal 1:** Improve dataset quality  
+- **Goal 2:** Fine-tune the model again and evaluate performance  
+- **Goal 3:** Work on mechanics for voice switching and personas inside Sugar  
+
+---
+
+## This Week’s Progress
+
+### **1. Improving the dataset**
+
+- The fine-tuning done using the previous dataset wasn’t up to expectations.  
+- So the next logical step was to refine the dataset and try again. And that’s exactly what I did.  
+- I added more diverse types of conversations and included a wider range of questions that kids might ask.  
+- I included cases where the child makes mistakes and the teacher responds by correcting them.  
+- I also added examples where the child makes spelling errors, and the teacher helps them fix those.  
+- Additionally, I included general conversational cases like “I love pizza” or “I just woke up” to make the dataset more natural.  
+
+### **2. Fine-tune the model again and evaluate performance**
+
+- I fine-tuned the [Llama 135M model](https://huggingface.co/amd/AMD-Llama-135m) again, this time on the [updated dataset](https://github.com/mebinthattil/AMD_Llama_135M). 
+- After fine-tuning, I asked the model a few questions to see how it performed.  
+- The results still weren’t up to expectations.  
```

Comment:
Again, your bullets go from something you did (e.g. I fine-tuned...) to something that happened (e.g. the results weren't good).

I bullet should be a list of similar information. Otherwise, you might want a paragraph instead. Or you have a couple sentences and then a list of "what I did".